Formula/libtrace.rb

```diff
@@ -1,14 +1,9 @@
 class Libtrace < Formula
   desc "Library for trace processing supporting multiple inputs"
-  homepage "https://research.wand.net.nz/software/libtrace.php"
-  url "https://research.wand.net.nz/software/libtrace/libtrace-4.0.19.tar.bz2"
-  sha256 "d1a2d756d744c4ffad480f6d4b0bdaca05a9966c87e491992c42007a22317e5a"
-  license "GPL-3.0-or-later"
-
-  livecheck do
-    url :homepage
-    regex(/href=.*?libtrace[._-]v?(\d+(?:\.\d+)+)\.t/i)
-  end
+  homepage "https://github.com/LibtraceTeam/libtrace"
+  url "https://github.com/LibtraceTeam/libtrace/archive/refs/tags/4.0.20-1.tar.gz"
+  sha256 "df6d9dc66d9e3a9e46a153e07e8d13967a327db547ce8ef3f2de691d7081344e"
+  license "LGPL-3.0-or-later"
 
```

Comment:
```suggestion

  livecheck do
    url :stable
    regex(/^v?(\d+(?:[.-]\d+)+)$/i)
  end
```

There are some tags in this repository that we'll want to avoid (e.g., `4.0.9-2-rpms`, `4.0.14-2-deb`, `4.0.14-2-rpm`), so we still need a `livecheck` block after these changes.